code/_helpers/turfs.dm

```diff
@@ -211,3 +211,38 @@
 		return list(new_turf, source)
 
 	return new_turf
+
+/proc/get_dir_z_text(turf/source, turf/target)
```

Comment:
this won't either, will it? if you're 15 tiles to the west of the bell on one z level, source.x < target.x, and so it'll say the bell is coming from the east, which is correct. but if you're 15 tiles to the west of the bell with a z-level transition in between, source.x > target.x, because you end up on the right side of the new level, so it'll say it's from the west which is wrong